src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,14 @@
+html {
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+}
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Don't reset `margin` and `padding` using `*` selector
You can forget about it and it can cause unexpected behavior
If you want to reset some value you need to set it directly to some tag

Reply:
missed that, thanks for the tip ;-) 🔥